protos/sigstore_rekor.proto

```diff
@@ -104,8 +104,10 @@ message TransparencyLogEntry {
         int64 integrated_time = 4 [(google.api.field_behavior) = REQUIRED];
         // The inclusion promise/signed entry timestamp from the log.
         // Required for v0.1 bundles, and MUST be verified.
-        // Optional for >= v0.2 bundles, and SHOULD be verified when present.
-        // Also may be used as a signed timestamp.
+        // Optional for >= v0.2 bundles if another source of signed time
+        // is present.
```

Comment:
One could interpret "another source of ... time" to be current time, but "current time" wouldn't be signed. We could be very prescriptive and say "When verifying long-lived certificates, the client MAY choose to not require a signed timestamp and instead use the system clock." Thoughts?

Reply:
That sounds good to me! Updating

Reply:
Updated in https://github.com/sigstore/protobuf-specs/pull/380/commits/d9edb63b1146eb1ef96dcebb3fd38ee1cef585aa -- it now talks about a "suitable" source of time, which can be either another signed time source *or* the current system time when the certs are long-lived. LMKWYT!